flow/CMakeLists.txt

```diff
@@ -161,53 +160,68 @@ if(WIN32)
   add_dependencies(flow_sampling_actors flow_actors)
 endif()
 
+# When creating a static or shared library, undefined symbols will be ignored.
```

Comment:
a nit: Thanks for adding this comment, may be we can copy/paste the same in the .cpp file as well?